jwql/jwql_monitors/monitor_filesystem.py

```diff
@@ -93,13 +93,6 @@ def files_per_filter():
         n_files[instrument] = {}
         for fname in FILTERS_PER_INSTRUMENT[instrument]:  # note that this does not include pupil wheel-based filters
             obs = Observations.query_criteria(filters=fname, instrument_name=JWST_INSTRUMENT_NAMES_MIXEDCASE[instrument])
-            batch_size = 5
-            batches = [obs[i:i+batch_size] for i in range(0, len(obs), batch_size)]
-
-            obs_table = [Observations.get_product_list(batch) for batch in batches]
-            products = unique(vstack(obs_table), keys='productFilename')
-            filtered_products = Observations.filter_products(products, productType=["SCIENCE"], productSubGroupDescription=['UNCAL'], extension="fits")
-
```

Comment:
well this is an exciting removal

Reply:
This one confuses me. It seems like none of the variables here returns a number of files. Searching the codebase though, it looks like we never call this function at all?

Reply:
Yeah, this one is strange for sure. I removed one unused variable, then another would pop up, then I'd remove that one and so on until the function was useless. I don't know if we want to keep it around or just scrap it.

Reply:
If we can scrap it it would be great to.  I love scrapping unused code, its the best.